v4.14/graph.yaml

```diff
@@ -27,6 +27,15 @@ entries:
           - rhtas-operator.v1.0.2
           - rhtas-operator.v1.1.0
           - rhtas-operator.v1.1.1
+      - name: rhtas-operator.v1.2.1
+        replaces: rhtas-operator.v1.0.0
+        skips:
+          - rhtas-operator.v1.0.1
+          - rhtas-operator.v1.0.2
+          - rhtas-operator.v1.1.0
+          - rhtas-operator.v1.1.1
+          - rhtas-operator.v1.1.2
+          - rhtas-operator.v1.2.0
```

Comment:
If I am reading this correctly, this will add the rhtas-operator.v1.2.1 operator to the stable-v1.1 channel instead of the stable-v1.2 channel